docs/features/pouch_with_rich_container.md

```diff
@@ -65,7 +65,7 @@ PID   USER     TIME  COMMAND
 
 ### Using systemd or sbin-init
 
-In order to use systemd or /sbin/init to init container, please make sure install them on image.
+In order to use systemd or /sbin/init to init container, please make sure install them on image. Please using pouch version not lower than 1.1.0 if you want to use systemd rich mode.
```

Comment:
Typos: `please make sure to install` and `Please use pouch`.

Reply:
good catch